src/index.ts

```diff
@@ -267,23 +265,9 @@ prog
     opts.name = opts.name || appPackageJson.name;
     opts.input = await getInputs(opts.entry, appPackageJson.source);
     const buildConfigs = createBuildConfigs(opts);
+    await util.promisify(mkdirp)(resolveApp('dist'));
```

Comment:
I haven't fully understood why this line was originally inside the condition. Why it shouldn't be creating dist folder no matter of what format is chosen?